src/Umbraco.Infrastructure/Services/Implement/PackagingService.cs

```diff
@@ -157,14 +159,15 @@ public IEnumerable<InstalledPackage> GetAllInstalledPackages()
 
             if (!installedPackages.TryGetValue(package.PackageName, out InstalledPackage? installedPackage))
             {
-                installedPackage = new InstalledPackage {
+                installedPackage = new InstalledPackage
+                {
                     PackageName = package.PackageName,
-                    Version = string.IsNullOrEmpty(package.Version) ? "Unknown" : package.Version,
                 };
 
                 installedPackages.Add(package.PackageName, installedPackage);
             }
 
+            installedPackage.Version = string.IsNullOrEmpty(package.Version) ? "Unknown" : package.Version;
```

Comment:
The version was previously only set when it couldn't find the package in the list, so it would be empty when it was already added when populating the list from the package migration states in the database.

Reply:
And this is now also conditionally set, so the version is only shown when available (you won't see `Version: Unknown` in the dashboard anymore) 🎉